Content.Server/Database/ServerDbBase.cs

```diff
@@ -671,6 +671,34 @@ await foreach (var json in query.Select(log => log.Json).AsAsyncEnumerable())
 
         #endregion
 
+        #region Whitelist
+
+        public async Task<bool> GetWhitelistStatusAsync(Guid player)
+        {
+            await using var db = await GetDb();
+
+            var list = await db.DbContext.Whitelist.Where(w => w.UserId == player).ToListAsync();
+            return list.Count > 0;
+        }
+
+        public async Task AddToWhitelistAsync(Guid player)
+        {
+            await using var db = await GetDb();
+
+            db.DbContext.Whitelist.Add(new Whitelist { UserId = player });
+            await db.DbContext.SaveChangesAsync();
+        }
+
+        public async Task RemoveFromWhitelistAsync(Guid player)
+        {
+            await using var db = await GetDb();
+            var entry = await db.DbContext.Whitelist.SingleAsync(w => w.UserId == player);
+            db.DbContext.Whitelist.Remove(entry);
+            await db.DbContext.SaveChangesAsync();
+        }
```

Comment:
The code is a bit inconsistent in this regard but I would prefer if these took in `NetUserId` instead of `Guid`.